packages/components/src/components/page-overlay/page-overlay.tsx

```diff
@@ -9,7 +9,7 @@ type TPageOverlay = {
     id?: string;
     is_from_app?: boolean;
     is_open?: boolean;
-    onClickClose: (event: MouseEvent) => void;
+    onClickClose: undefined | ((event: MouseEvent) => void);
```

Comment:
why do we need undefined here?

Reply:
because I changed the `onClickClose` prop to be optional in `MobileFullPageModal` it was giving some eslint error for PageOverlay since it was undefined, so I added the typing here

Reply:
what if we make it optional also 🤔 

```suggestion
    onClickClose?: (event: MouseEvent) => void;
```

Reply:
let me check, i think I had some issue if i tried that